src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -371,16 +371,22 @@ export default class AircraftInstanceModel {
     }
 
     /**
+     * Returns a true value if there is a match from the callsignToMatch
+     *
      * @for AircraftInstanceModel
      * @method matchCallsign
      * @param callsign {string}
      */
     matchCallsign(callsignToMatch) {
-        if (callsignToMatch === '*') {
+        const shouldMatchAnyCallsign = callsignToMatch === '*';
+
+        // checks to see if the given call sign matches the airline Id + flight number
+        if (shouldMatchAnyCallsign || _isEqual(callsignToMatch.toUpperCase(), this.callsign)) {
```

Comment:
Couldn't we get away with simply `===` here? It would be more performant, and since we aren't comparing arrays, I don't think there's any real advantage to using lodash `_isEqual`. Same comment for the other line.